test/suites/isolateSubtrees.test.js

```diff
@@ -10,33 +10,45 @@ describe('isolateSubtrees', () => {
     // NOTE: in jest, the only display check that's expected to work is 'none' because
     //  jsDom doesn't support the APIs needed to make the other modes work
     trapOptions = {
-      isolateSubtrees: true,
       tabbableOptions: { displayCheck: 'none' },
     };
   });
 
+  /**
+   * INERT USAGE
+   */
```

Comment:
I don't think any of the tests are checking that `isolateSubtrees: 'inert'` also uses the `inert` attribute, same as `isolateSubtrees: true`. I think it would be worth one simple test just to check that expectation (that when the `'inert'` string value is used for the option, the result is that adjacent nodes are disabled using the `inert` attribute). Nothing more since the existing tests are covering all the details.